ksqldb-engine/src/test/java/io/confluent/ksql/physical/pull/HARoutingTest.java

```diff
@@ -674,13 +675,13 @@ public void forwardingError_cancelled() throws ExecutionException, InterruptedEx
   @Test
   public void forwardingError_noRows() {
     // Given:
-    locate(location2);
+    locate(location4);
```

Comment:
With the change where we keep trying standbys if a host throws exception, these tests were not failing :) So, had to use a location for a single node